editor/connections_dialog.cpp

```diff
@@ -662,12 +668,13 @@ void ConnectDialog::init(const ConnectionData &p_cd, const PackedStringArray &p_
 
 	bool b_deferred = (p_cd.flags & CONNECT_DEFERRED) == CONNECT_DEFERRED;
 	bool b_oneshot = (p_cd.flags & CONNECT_ONE_SHOT) == CONNECT_ONE_SHOT;
+	bool b_append_source = (p_cd.flags & CONNECT_APPEND_SOURCE_OBJECT) == CONNECT_APPEND_SOURCE_OBJECT;
```

Comment:
```suggestion
	bool b_append_source = (p_cd.flags & CONNECT_APPEND_SOURCE_OBJECT);
```
The comparison is unnecessary.